tm2/pkg/bft/abci/types/types.go

```diff
@@ -116,6 +117,32 @@ func (r ResponseBase) IsErr() bool {
 	return r.Error != nil
 }
 
+func (r ResponseBase) EventString() string {
```

Comment:
```suggestion
func (r ResponseBase) String() string {
```
What about removing the intermediary step?

Reply:
I gave it a shot, but I couldn't figure out how to approach it honestly.